services/ui-src/src/App.js

```diff
@@ -60,12 +89,7 @@ function App() {
                 </>
               ) : (
                 <>
-                  <LinkContainer to="/signup">
```

Comment:
Signup is now handled by the cognito hosted ui, if applicable.
Okta signup is not supported or considered.